pkg/devicemapper/devmapper.go

```diff
@@ -342,7 +341,7 @@ func RemoveDevice(name string) error {
 		return fmt.Errorf("devicemapper: Error running RemoveDevice %s", err)
 	}
 
-	return nil
+	return UdevWait(cookie)
```

Comment:
If you are waiting inline for UdevWait(), do we still need to move cookie to heap? Looks like stack will still be around as we are waiting for UdevWait() inline.